tests/unit/amo/pages/TestSearchTools.js

```diff
@@ -55,33 +53,4 @@ describe(__filename, () => {
 
     expect(root.find(HeadLinks)).toHaveLength(1);
   });
-
-  describe('enableFeatureRemoveSearchTools = true', () => {
-    it('sends a server redirect', () => {
```

Comment:
We need to keep this test case all the time now. I suppose you can remove the `describe` block above, though, and only keep/rename `it('sends a server redirect to support old search tool URLs')`

Reply:
@willdurand Will I do the same for other tests above ? Or just only this one ?